src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -51,6 +53,12 @@
 __all__ = ["TextGenerationPipeline"]
 
 
+class FinishReason(Enum):
+    STOP = "stop token generated."
+    LENGTH = "max token generation reached."
+    TIME = "max time reached."
```

Comment:
I would advocate for making these more terse i.e. simply `stop`, `length`, `time` or matching the OpenAI Completions API https://platform.openai.com/docs/guides/gpt/chat-completions-response-format